sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/context/PreferenceTokenStorage.java

```diff
@@ -0,0 +1,57 @@
+package com.microsoft.appcenter.utils.context;
+
+import android.content.Context;
+
+import com.microsoft.appcenter.utils.crypto.CryptoUtils;
+import com.microsoft.appcenter.utils.storage.SharedPreferencesManager;
+
+/**
+ * Storage for tokens that uses Shared Preferences. Handles saving and encryption.
+ */
+public class PreferenceTokenStorage implements TokenStorage {
+
+    /**
+     * Context instance.
+     */
+    private final Context mContext;
+
+    /**
+     * Default constructor.
+     *
+     * @param context context instance.
+     */
+    public PreferenceTokenStorage(Context context) {
+        mContext = context;
+    }
+
+    /**
+     * Base key for stored preferences.
+     */
+    static final String PREFERENCE_PREFIX = "AppCenter.";
```

Comment:
This doesn't look like it's used anywhere else.  I would prefer that this just be combined with `PREFERENCE_KEY_AUTH_TOKEN`

Reply:
@jwallra `PREFERENCE_PREFIX` is used below in `PREFERENCE_KEY_AUTH_TOKEN`